front/pages/api/w/[wId]/vaults/[vId]/contentHandler.ts

```diff
@@ -0,0 +1,83 @@
+import type { ContentNodeType, WithAPIErrorResponse } from "@dust-tt/types";
```

Comment:
It should be `content-handler` if it's an endpoint, otherwise it should live in `front/api/lib`.